FRAuth/FRAuth/Device/ProfileCollector.swift

```diff
@@ -19,7 +19,7 @@ public class ProfileCollector: DeviceCollector {
     public var collectors: [DeviceCollector]
     
     /// Private initialization method which initializes default array of ProfileCollector
```

Comment:
Minor: "Private" should become "public" in comment

Reply:
Thanks Stoyan! Updated the comment accordingly.